LOMS/src/render/mouse.js

```diff
@@ -1,17 +1,27 @@
-const MOUSE_TEXTURE_PATH = '../../assets/mouse.png';
+const MOUSE_DEFAULT_TEXTURE_PATH = '../../assets/mouse.png';
 const MOUSE_UP_TEXTURE_PATH = '../../assets/mouseUp.png';
+const MOUSE_DOWN_TEXTURE_PATH = '../../assets/mouseDown.png';
+const MOUSE_LEFT_TEXTURE_PATH = '../../assets/mouseLeft.png';
+const MOUSE_RIGHT_TEXTURE_PATH = '../../assets/mouseRight.png';
+
 
 export default class Mouse {
-    constructor(){
+    EDGE = 40;
```

Comment:
I think this variable does not need to be a member of Mouse, and why did you write EDGE in prototype but excluded mouse texture variables?

Reply:
No, it doesn't. I put it there just in case you want it to be dynamic.  

I moved mouse texture vars mainly for 2 reasons:
1. They load the image, so I assume there's side-effect. 
2. It was part of my attempt to write tests. If you leave them there, there's no way to test this class without `PIXI`. Although this still doesn't support testing without `PIXI`.

I can put the vars in prototype, and then in `init` check existence and initialize them only once. But, I think, `Mouse` should only be instantiated once anyway, so it won't cost if they are not in the prototype.

Reply:
Wait, actually they are not in prototype. This is the new field declarations syntax, and the declared vars are not shared among instances.

Reply:
This PR is okay to merge, just make sure next PR with unit test and also without pixi dependency